core/chains/evm/config/config.go

```diff
@@ -80,6 +73,26 @@ type ChainScopedOnlyConfig interface {
 
 	// OCR2 chain specific config
 	OCR2AutomationGasLimit() uint32
+
+	EVM() EVM
+}
+
+type EVM interface {
```

Comment:
This seems odd - do we need this extra indirection? IIRC `ChainScopedOnlyConfig` is already only EVM, without any global/root config to distinguish from.

Reply:
Oh could we have this method on `ChainScopedConfig` instead? That seems like a cleaner separation :thinking:

Reply:
Yeah this was the bit I wasn't super sure about. My reason for moving everything EVM-related to cfg.EVM() was because that's the config key everything is under.

I can move it to ChainScopedConfig or remove it entirely if you'd rather go that way? (I'd probably prefer to move it since I do think there's some sense to having the EVM keys under `cfg.EVM()`, but don't have super strong feelings either way)

Reply:
(I've moved to ChainScopedConfig but lmk if you'd rather I removed entirely)

Reply:
Sounds good. Yeah I see that `.Evm()` makes sense at the callsite with the composed interface :+1: